examples/Reference Building - Primary School.xml

```diff
@@ -180,6 +180,7 @@
                           </OccupancyScheduleIDs>
                           <OccupantsActivityLevel>Low</OccupantsActivityLevel>
                           <DaylightingIlluminanceSetpoint>500</DaylightingIlluminanceSetpoint>
```

Comment:
This value is clearly in Lux and should be 50 ft at most (https://www.lightingdesignlab.com/sites/default/files/pdf/Footcandle_Lighting%20Guide_Rev.072013.pdf)

Reply:
@nllong I added the "units" element. The value was already present in the example file. Should we change it?

Reply:
yeah, we'll need to update the example files. Basically divide the values > 100 by 10 (10.76 to be precise).